src/ScaleFactor.cc

```diff
@@ -68,37 +68,33 @@ void ScaleFactor::init_EG_ScaleFactor(TString inputRootFile, bool isTriggerSF) {
     double data_pt_nom[nbin_pt] = {0};
     double data_eff_nom[nbin_pt] = {0};
     double data_pt_errlow[nbin_pt] = {0};
-    double data_eff_errlow[nbin_pt] = {0};
     double data_pt_errhigh[nbin_pt] = {0};
-    double data_eff_errhigh[nbin_pt] = {0};
+    double data_eff_err[nbin_pt] = {0};
     double mc_pt_nom[nbin_pt] = {0};
     double mc_eff_nom[nbin_pt] = {0};
     double mc_pt_errlow[nbin_pt] = {0};
-    double mc_eff_errlow[nbin_pt] = {0};
     double mc_pt_errhigh[nbin_pt] = {0};
-    double mc_eff_errhigh[nbin_pt] = {0};
+    double mc_eff_err[nbin_pt] = {0};
```

Comment:
> Motivated by previously found bugs: are we sure these are _always_ initialized to the nominal values?

not sure what exactly you mean here



> Also, doubles should be initialized to `0.` instead of `0` for readibility, but it has no effect.

will change that